04_numeronym/patrickmarabeas/main.go

```diff
@@ -0,0 +1,36 @@
+package main
+
+import (
+	"fmt"
+	"io"
+	"os"
+)
+
+var out io.Writer = os.Stdout
+
+// Convert string to a numeronym
+func generateNumeronym(s string) string {
+	length := len(s)
```

Comment:
length variable creation may be unnecessary as its used in only one case.

So perhaps `if len(s)<4` would suffice.

Reply:
It's also used in the return statement twice.